komga-webui/src/locales/en.json

```diff
@@ -397,7 +397,7 @@
       "tab_general": "General",
       "tab_metadata": "Metadata",
       "tab_options": "Options",
-      "tooltip_scanner_force_modified_time": "Enable if the library is on a Google Drive",
```

Comment:
"Enable" only needed for something that can be on, but yet disabled.
The distinction also has to be meaningful to the user to introduce the more difficult word "enable" IMO.